apache/trafficcontrol
# Remove old E2E TP Test

<hr>

## Which Traffic Control components are affected by this PR?

- Documentation
- Traffic Portal
- CDN in a Box


## What is the best way to verify this PR?

1. Make sure old E2E Traffic Portal completely gone from open source. 
2. Make sure nothing break

## If this is a bugfix, which Traffic Control versions contained the bug?



## PR submission checklist
